packages/make-styles/src/runtime/resolveStyleRules.ts

```diff
@@ -74,6 +93,7 @@ export function resolveStyleRules(
           }
         : undefined;
 
```

Comment:
Not directly related to your changes, but `rtlClassName` should be computed only if `flippedInRtl`

Reply:
Did I correctly get your suggestion (3d97685)?